callosum/__init__.py

```diff
@@ -3,17 +3,36 @@
     RPCMessage, RPCMessageTypes,
 )
 from .eventmessage import (
-    EventMessage, EventTypes,
+    EventMessage,
+    EventTypes,
+    EventHandler,
+)
+from .peer import (
+    Publisher,
+    Subscriber,
+    Peer,
+)
+from .exceptions import (
+    CallosumError,
+    RedisAddressParamError,
```

Comment:
`ParamError` was too generic but `RedisAddressParamError` is too specific.
The redis lower layer may define its own specialized exceptions classes, but in the top level of Callosum, I would like to keep only the base exceptions that could be specialized further in submodules and subpackages.
How about renaming this as `InvalidAddressError` to embrace any errors due to invalid address values?

In general, naming is a crucial part of building abstractions and we must find a balance between generality and specificity.
The exception names may represent either a choice or combinations of them, among the source, action, and condition of errors.
For instance, `CallosumError` embraces all exceptions whose source is the Callosum library.
`ClientError` embraces all exceptions originated from the client-side.
But for more specific errors like `RedisAddressParamError`, I believe it is better to include specific condition (e.g., "Invalid") while the base error classes should embrace the superset of potential sources (e.g., "any address values" -> "Address"). Combining these, my suggestion is `InvalidAddressError`.
Inside the redis lower layer, we could further specialize this exception like `InvalidRedisBinderAddressError`, but I don't think this is necessary for our case.

Reply:
I corrected it. Is it fine now?